scripts/update/Updater.php

```diff
@@ -44,8 +44,10 @@ public function update($initialVersion)
             AclProxy::applyRule(new AccessRule('grant', 'http://www.tao.lu/Ontologies/generis.rdf#AnonymousRole', ['ext'=>'taoOpenId','mod' => 'Connect', 'act' => 'callback']));
             $this->setVersion('0.0.3');
         }
-        
-        if ($this->isVersion('0.0.3')) {
+
+        $this->skip('0.0.3', '0.1.0');
```

Comment:
modification of past update scripts should only be done in bug fixes that fix issues with update scripts